examples/gallery/plot/colorbar.py

```diff
@@ -49,7 +49,8 @@
     # with a length/width (+w) of 7cm by 0.5cm and a box for NaN values (+n)
     position="JMR+o1c/0c+w7c/0.5c+n+mc",
```

Comment:
In L6, change cmap argument to cmap parameter

Reply:
In L7, change position argument to position parameter

Reply:
In L21, change ``(BL)`` to ``(**BL**)``

Reply:
Fixed.